crates/nu-std/std/mod.nu

```diff
@@ -142,6 +142,7 @@ export def clip [
     --no-notify: bool  # do not throw a notification (only on linux)
     --no-strip: bool  # do not strip ANSI escape sequences from a string
     --expand (-e): bool  # auto-expand the data given as input
+    --charpage (-c): int  # the id of the charpage to use (only on Windows), see https://en.wikipedia.org/wiki/Windows_code_page, e.g. 65001 is for UTF-8
```

Comment:
It's called a `codepage` on Windows.

Reply:
woopsie :eyes: 

should be better in f6db72a47 :wink: